up42/storage.py

```diff
@@ -141,7 +141,7 @@ def get_assets(
             producer_names: Search for assets from any of the provided producers.
             tags: Search for assets with any of the provided tags.
             sources: Search for assets from any of the provided sources.\
-                The allowed values: `"ARCHIVE"`, `"TASKING"`, `"ANALYTICS"`, `"USER"`.
+                The allowed values: `"ARCHIVE"`, `"TASKING"`, `"USER"`.
```

Comment:
oh, wow! I failed to see that there is no `PROCESSING` source in that list.
Still, we shouldn't be adding it here since the API will not return any assets with the `PROCESSING` source on the list assets endpoint